src/playbook.yaml

```diff
@@ -47,16 +47,6 @@
           echo 'LoadModule python_module /usr/lib/apache2/modules/mod_python.so' >> /etc/apache2/mods-available/python.load
         chdir: /tmp/mod_python_source
 
-    - name: Install python3.8
```

Comment:
why did you remove python 3.8 ?

Reply:
I think it's causing cloud-init to fail. As-is, the instance does not deploy (it times out and if I prevent the delete it is not correctly initialized (i.e. does not pass one of the reachability tests)).

Reply:
that doesn't make sense because it does not fail on the v1.2.8 version which has python 3.10.12